src/lexer.rs

```diff
@@ -276,7 +378,7 @@ impl JavascriptLexer {
       },
       ImportToken::StatementEnd => {
         match self.current_char {
-          c if c.is_whitespace() => {},
+          c if c == ' ' => {},
```

Comment:
Good catch - new line characters need to be treated differently than space characters here.